tsconfig.json

```diff
@@ -8,8 +8,15 @@
       "sass": ["sass/"]
     },
     "rootDirs": ["public/"],
-    "typeRoots": ["node_modules/@types", "public/app/types"]
+    "typeRoots": ["node_modules/@types", "public/app/types"],
+    "allowJs": true
   },
   "extends": "@grafana/tsconfig/base.json",
-  "include": ["public/app/**/*.ts*", "public/e2e-test/**/*.ts", "public/test/**/*.ts", "public/vendor/**/*.ts"]
+  "include": [
+    "public/app/**/*.ts*",
+    "public/e2e-test/**/*.ts",
+    "public/test/**/*.ts",
+    "public/vendor/**/*.ts",
+    "packages/jaeger-ui-components/typings"
```

Comment:
For this to work I needed to include it also here, including it in only the packages tsconfig wasn't enough